vendor.conf

```diff
@@ -113,3 +113,9 @@ github.com/facebookgo/pidfile f242e2999868dcd267a2b86e49ce1f9cf9e15b16
 github.com/facebookgo/atomicfile 2de1f203e7d5e386a6833233882782932729f27e
 github.com/cpuguy83/go-md2man 1d903dcb749992f3741d744c0f8376b4bd7eb3e1
 github.com/russross/blackfriday 6d1ef893fcb01b4f50cb6e57ed7df3e2e627b6b2
+github.com/prometheus/client_golang v0.9.0-pre1
+github.com/prometheus/common 2e54d0b93cba2fd133edc32211dcc32c06ef72ca
```

Comment:
It was really surprising but the prometheus internal dependencies seemed to not have versions that matched the client. I'm not sure how best to track this but I picked master on the day I committed this :/